src/crowdin_gui.cpp

```diff
@@ -676,6 +677,19 @@ bool ExtractCrowdinMetadata(CatalogPtr cat,
                 : cat->GetLanguage();
     }
 
+    const auto xliff = std::dynamic_pointer_cast<XLIFFCatalog>(cat);
+    if (xliff)
+    {
+        if (std::strcmp(xliff->GetXPathValue("file/header/tool//@tool-id"), "crowdin") == 0)
```

Comment:
I must be missing something, why is `tool@tool-id` not working?

I'd rather `GetXPathValue` returned `std::string` for easy comparison of the return value. If returning `const char*`, it definitely should return `nullptr` and not `""` on not found.

Reply:
> I must be missing something, why is `tool@tool-id` not working?

Yep, it does not (tried) since needs full path to node as well as `/` before `@` (single `/` is enough, not double, corrected in 8767181)

> I'd rather `GetXPathValue` returned `std::string` for easy comparison of the return value. If returning `const char*`, it definitely should return `nullptr` and not `""` on not found.

Done 8767181